perfil/company/models.py

```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
I do not really see the point of creating a file just for putting a comment on it. I guess you should create a file when you really need it. That means when you have a broken test which tells you that you need some file or piece of code.

Reply:
It's auto created by Django. Doesn't harm performance or development as far as I got. So I see no urge to clean up ; )